### PR TITLE
miscellaneous-functions: Update reference manual

### PR DESCRIPTION
**Pull request by qiancai**
### What is changed, added or deleted? (Required)

Closes #16816

### Which TiDB version(s) do your changes apply to? (Required)

**Tips for choosing the affected version(s):**

By default, **CHOOSE MASTER ONLY** so your changes will be applied to the next TiDB major or minor releases. If your PR involves a product feature behavior change or a compatibility change, **CHOOSE THE AFFECTED RELEASE BRANCH(ES) AND MASTER**.

For details, see [tips for choosing the affected versions](https://github.com/pingcap/docs/blob/master/CONTRIBUTING.md#guideline-for-choosing-the-affected-versions).

- [x] master (the latest development version)
- [x] v8.0 (TiDB 8.0 versions)
- [ ] v7.6 (TiDB 7.6 versions)
- [ ] v7.5 (TiDB 7.5 versions)
- [ ] v7.4 (TiDB 7.4 versions)
- [ ] v7.1 (TiDB 7.1 versions)
- [ ] v6.5 (TiDB 6.5 versions)
- [ ] v6.1 (TiDB 6.1 versions)
- [ ] v5.4 (TiDB 5.4 versions)
- [ ] v5.3 (TiDB 5.3 versions)
- [ ] v5.2 (TiDB 5.2 versions)
- [ ] v5.1 (TiDB 5.1 versions)
- [ ] v5.0 (TiDB 5.0 versions)

### What is the related PR or file link(s)?

- This PR is translated from: https://github.com/pingcap/docs/pull/16855
- Other reference link(s):

### Do your changes match any of the following descriptions?

- [ ] Delete files
- [ ] Change aliases
- [ ] Need modification after applied to another branch 
- [ ] Might cause conflicts after applied to another branch

### CODE REVIEW

File: functions-and-operators/miscellaneous-functions.md [hunk 1]
```diff
@@ -11,27 +11,410 @@ TiDB 支持使用 MySQL 5.7 中提供的大部分[其他函数](https://dev.mysq
 ## 支持的函数
```
Comment (Oreoxmt) [inline functions-and-operators/miscellaneous-functions.md:11]: L4:
```
summary: TiDB 支持大部分 MySQL 8.0 中的其他函数。
```

L9:

```
TiDB 支持使用 MySQL 8.0 中提供的大部分[其他函数](https://dev.mysql.com/doc/refman/8.0/en/miscellaneous-functions.html)。
```

File: functions-and-operators/miscellaneous-functions.md [hunk 2]
```diff
@@ -11,27 +11,410 @@ TiDB 支持使用 MySQL 5.7 中提供的大部分[其他函数](https://dev.mysq
...
+| SLEEP(1.5) |
++------------+
+|          0 |
++------------+
+1 row in set (1.50 sec)
+```
+
+### UUID()
+
+`UUID()` 函数用于返回通用唯一标识符（UUID）(version 1)。UUID 的定义可参考 [RFC 4122](https://datatracker.ietf.org/doc/html/rfc4122) 。
```
Comment (Oreoxmt) [inline functions-and-operators/miscellaneous-functions.md:366]:
```suggestion
`UUID()` 函数用于返回通用唯一标识符 (UUID) version 1。UUID 的定义可参考 [RFC 4122](https://datatracker.ietf.org/doc/html/rfc4122)。
```

**Review by Oreoxmt — APPROVED** [review]

File: functions-and-operators/miscellaneous-functions.md [hunk 3]
```diff
@@ -1,37 +1,420 @@
 ---
 title: 其他函数
 aliases: ['/docs-cn/dev/functions-and-operators/miscellaneous-functions/','/docs-cn/dev/reference/sql/functions-and-operators/miscellaneous-functions/']
-summary: TiDB 支持大部分 MySQL 5.7 中的其他函数，包括 ANY_VALUE()、BIN_TO_UUID()、DEFAULT()、INET_ATON()、INET_NTOA()、INET6_ATON()、INET6_NTOA()、IS_IPV4()、IS_IPV4_COMPAT()、IS_IPV4_MAPPED()、IS_IPV6()、NAME_CONST()、SLEEP()、UUID()、UUID_TO_BIN()和VALUES()。不支持的函数有 UUID_SHORT() 和 MASTER_WAIT_POS()。
+summary: TiDB 支持大部分 MySQL 8.0 中的其他函数，包括 ANY_VALUE()、BIN_TO_UUID()、DEFAULT()、INET_ATON()、INET_NTOA()、INET6_ATON()、INET6_NTOA()、IS_IPV4()、IS_IPV4_COMPAT()、IS_IPV4_MAPPED()、IS_IPV6()、NAME_CONST()、SLEEP()、UUID()、UUID_TO_BIN()和VALUES()。不支持的函数有 UUID_SHORT() 和 MASTER_WAIT_POS()。
```
Comment (qiancai) [inline functions-and-operators/miscellaneous-functions.md:4]:
```suggestion
summary: TiDB 支持大部分 MySQL 8.0 中的其他函数。
```

File: functions-and-operators/miscellaneous-functions.md [hunk 4]
```diff
@@ -1,37 +1,420 @@
 ---
 title: 其他函数
 aliases: ['/docs-cn/dev/functions-and-operators/miscellaneous-functions/','/docs-cn/dev/reference/sql/functions-and-operators/miscellaneous-functions/']
-summary: TiDB 支持大部分 MySQL 5.7 中的其他函数，包括 ANY_VALUE()、BIN_TO_UUID()、DEFAULT()、INET_ATON()、INET_NTOA()、INET6_ATON()、INET6_NTOA()、IS_IPV4()、IS_IPV4_COMPAT()、IS_IPV4_MAPPED()、IS_IPV6()、NAME_CONST()、SLEEP()、UUID()、UUID_TO_BIN()和VALUES()。不支持的函数有 UUID_SHORT() 和 MASTER_WAIT_POS()。
+summary: 支持使用 MySQL 8.0 中提供的大部分其他函数。
```
Comment (qiancai) [inline functions-and-operators/miscellaneous-functions.md:4]:
```suggestion
summary: TiDB 支持使用 MySQL 8.0 中提供的大部分其他函数。
```

**Pull request merged**